--- a/week10/exercise_templates/w10_template_03_ConnectFour/.gitignore
+++ b/week10/exercise_templates/w10_template_03_ConnectFour/.gitignore
@@ -1,2 +1,3 @@
 /Debug (Linux)/
 /Debug (Windows)/
+/Release/
--- a/week10/exercise_templates/w10_template_03_ConnectFour/ConnectFourWindow.cpp
+++ b/week10/exercise_templates/w10_template_03_ConnectFour/ConnectFourWindow.cpp
@@ -10,6 +10,7 @@
 #include <string>
 
 void ConnectFourWindow::run() {
+	window.setFramerateLimit(60);
 	while (window.isOpen()) {
 		checkConnection();
 		handleEvents();
--- a/week10/exercise_templates/w10_template_03_ConnectFour/ConnectFourWindow.h
+++ b/week10/exercise_templates/w10_template_03_ConnectFour/ConnectFourWindow.h
@@ -61,8 +61,14 @@ private:
 		}
 		return font;
 	}() };
-	sf::RenderWindow window { sf::VideoMode { 800, 710 }, "Connect Four (" + controller.gameState().peerState.name + ")", sf::Style::Default,
-			sf::ContextSettings { 0, 0, antiAliasing } };
+
+	sf::RenderWindow window {
+		sf::VideoMode { 800, 710 },
+		"Connect Four (" + controller.gameState().peerState.name + ")",
+		sf::Style::Default,
+		sf::ContextSettings { 0, 0, antiAliasing }
+	};
+
 	inline static std::array<sf::Color, 3> const colors { c4::Color::White, c4::Color::Red, c4::Color::Yellow };
 	inline static sf::Color const latestBorderColor { c4::Color::DarkerBlue };
 	constexpr static unsigned antiAliasing = 8;
